### PR TITLE
🐛 Fix CLI output on Windows

### LINKED ISSUE #211: Windows scan output formatting is not working ...
**Describe the bug**
Output from `cnspec scan local` in PowerShell is not working ...

Screen Shot 2022-11-25 at 11 14 08 AM

I tested this in both AWS and GCP with WIndows 2019

### PR DESCRIPTION
**Pull request by czunker**
Fixes #211

Signed-off-by: Christian Zunker <christian@mondoo.com>

### CODE REVIEW

**Comment by czunker** [comment]
Windows output now looks like expected:

Screenshot from 2022-12-01 14-24-45

File: cli/reporter/print_windows.go [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package reporter
+
+const NewLineCharacter = "\r\n"
```
Comment (vjeffrey) [inline cli/reporter/print_windows.go:3]: nice catch!

File: cli/reporter/print_compact.go [hunk 2]
```diff
@@ -360,47 +360,52 @@ func (r *defaultReporter) printControl(score *policy.Score, query *policy.Mquery
...
+		r.out.Write([]byte(NewLineCharacter))
 
 	case policy.ScoreType_Result:
 		r.out.Write([]byte(r.printScore(title, score, query)))
 
 		// additional information about the failed query
 		if !r.isCompact && score.Value != 100 {
-			r.out.Write([]byte("  Query:\n" + stringx.Indent(4, query.Query)))
-			r.out.Write([]byte{'\n'})
+			queryString := strings.ReplaceAll(stringx.Indent(4, query.Query), "\n", NewLineCharacter)
```
Comment (czunker) [inline cli/reporter/print_compact.go:384]: `stringx.Indent` adds a newline. So, I have to replace them after the indent.

**Comment by czunker** [comment]
`-o full` output is now also fixed.

**Review by vjeffrey — APPROVED** [review]
thank you!

**Pull request merged**